components/results/ResultsLoading.tsx

```diff
@@ -0,0 +1,22 @@
+import CenteredCardContent from '../common/CenteredCardContent';
+
+export default function ResultsLoading({ willTakeLonger }: { willTakeLonger: boolean }): JSX.Element {
+  return (
+    <CenteredCardContent
+      title=""
+      iconClasses=""
+      isLoading
+      description={
+        willTakeLonger ? (
+          <p>
+            <strong>This verification takes more time than usual.</strong>
```

Comment:
Change to:

> This identity verification is taking more time than usual.

Reply:
Good catch I am ashamed that I wrote that so wrong 😅
I guess I didn't re-read what I wrote. Thanks

Reply:
I'm always impressed by you (and others) who speak multiple languages. It's amazing. I certainly wouldn't be embarrassed.